### ISSUE TITLE
Clarify how `align-` properties work with grid

### ISSUE THREAD

**Issue by zombiezen** [tailwindlabs/tailwindcss.com#1698]
Hey folks, I really appreciate the great docs on this project, great work!

I had a bit of confusion while porting over some existing CSS to use Tailwind that would use `align-self: start` for a grid cell. I expected there to be a 1:1 mapping in the [Align Self classes](https://tailwindcss.com/docs/align-self), but I was confused that `self-start` would expand to `align-self: flex-start` and I couldn't set arbitrary values. Further, the docs in the [Start section](https://tailwindcss.com/docs/align-self) only talk about how the property behaves in a Flex context.

Upon doing further research, I think I'm understanding correctly that [using `flex-start` behaves identically to `start`](https://drafts.csswg.org/css-align/#valdef-self-position-flex-start), but I think it would be helpful for others to spell this out explicitly in the docs. WDYT?

(If you do decide to clarify this, I think the [Justify Content](https://tailwindcss.com/docs/justify-content), [Align Content](https://tailwindcss.com/docs/align-content), and [Align Items](https://tailwindcss.com/docs/align-items) pages would also benefit from this clarification.)

**Comment by adamwathan** [comment]
Hey! Yeah as you've discovered `flex-start` behaves the same as `start` outside of flexbox contexts. I just spent 20 minutes trying to find _any_ scenario where `flex-start` and `start` behave differently and I can't seem to find one.

When we added these utilities to Tailwind `align-self: start` didn't have good browser support (Safari only got support in 15.4) which is why we didn't even bother with it.

I'm not sure what specifically we could do differently here in the docs without the risk of making things too verbose and hard to skim, but if this proves to be a common struggling point we can definitely look into making some tweaks.

Going to close as we try to reserve issues for very actionable bugs but appreciate the suggestion regardless!
